dev/TEAL_opcodes.md

```diff
@@ -1128,3 +1229,74 @@ bitlen interprets arrays as big-endian integers, unlike setbit/getbit
 - Pushes: []byte
 - push a byte-array of length X, containing all zero bytes
 - LogicSigVersion >= 4
+
+## log
+
+- Opcode: 0xb0
+- Pops: *... stack*, []byte
+- Pushes: _None_
+- write bytes to log state of the current application
+- LogicSigVersion >= 5
+- Mode: Application
+
+`log` can be called up to MaxLogCalls times in a program, and log up to a total of 1k bytes.
+
+## tx_begin
```

Comment:
What happens if you begin a transaction and never `tx_submit` it?
What happens if you begin a 17-th transaction (while only 16 transactions are allowed).

Reply:
never submit? nothing. You can still pass with a transaction "prepared" but not submitted.
begin 17th? nothing. The check is in tx_submit

I'll put in some wording in the opcode docs.